packages/manager/apps/account-disable-2fa/src/components/FileInput/FileInput.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FunctionComponent, useContext, useRef } from 'react';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  ODS_BUTTON_SIZE,
+  ODS_BUTTON_TYPE,
+  ODS_ICON_NAME,
+  ODS_ICON_SIZE,
+} from '@ovhcloud/ods-components';
+import { OsdsButton, OsdsIcon } from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { FileInputContext } from './FileInputContainer';
+
+type Props = {
+  className?: string;
+};
+
+export const FileInput: FunctionComponent<Props> = ({ className }) => {
+  const context = useContext(FileInputContext);
+  if (!context) {
+    throw Error(
+      'The component <FileInput /> must be a child of FileInputContainer',
+    );
+  }
```

Comment:
Is it necessary ? `useContext` won't throw before if there is no context found ?

Reply:
Yeah, useContext doesn't throw an exception if the component is outside of the provider, useContext simply returns an undefined context.